qiskit_ibm_provider/transpiler/passes/scheduling/__init__.py

```diff
@@ -32,6 +32,13 @@
 Below we demonstrate how to schedule and pad a teleportation circuit with delays
 for a dynamic circuit backend's execution model:
 
+.. jupyter-execute::
+    :hide-code:
+    :hide-output:
+
+    import warnings
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+
```

Comment:
The only way I found to make the docs build pass was to filter the deprecation warning raised in 0.45 after the try-except. It's not ideal to have the try-except in an example, but these examples are run. An alternative would be to skip the run, in that case we wouldn't need the try-except or the filter.